spec/site_prism/element_spec.rb

```diff
@@ -31,6 +31,16 @@
         expect(subject).not_to have_element_two
       end
 
+      it 'raises a warning when the name starts with no_' do
+        log_messages = capture_stdout do
+          SitePrism.log_level = :WARN
+          expect { subject.no_such_element }
+          .to raise_error(Capybara::ElementNotFound)
+        end
+        expect(lines(log_messages)).to eq 3
+        expect(log_messages).to include("should not","start","with no_")
```

Comment:
Message validation is not required here, because we already have tests on the deprecator that it passes messages through. Here we are checking that Warning log messages are generated, that is all.